### ISSUE TITLE
Learn More in the form of a button & Improvisations

### ISSUE THREAD

**Issue by riya-rjha** [Innovateninjas/Paws-frontend#463]
![Screenshot (54)](https://github.com/Innovateninjas/Paws-frontend/assets/112813772/ef7a7729-b4f0-47f7-8a1f-581b65753567)

Here the text can be transformed into the part and after that a few dots like the desired text and then ...
Plus the Learn more can be added like a button under the text with UI similar to the UI of the website. 

Kindly assign me this issue under GSSoC.

**Comment by codewarnab** [comment]
add screenshot which example  are talking about

**Comment by riya-rjha** [comment]
![Screenshot (68)](https://github.com/Innovateninjas/Paws-frontend/assets/112813772/d7fcbc35-34c4-4750-a41d-0c2a899c5aec)

How about this? Of course this is just a demo, the real one would be with hover effects & other properties too that would fit well within the website composition.

**Comment by codewarnab** [comment]
tbh not looking good but you can  draw the design in figma or code it show me if i like it give a pr else not ...

**Comment by codewarnab** [comment]
work if you are ok with that

**Comment by riya-rjha** [comment]
Okay will do one thing, will design it on Figma & then add here the screenshot. If it works well then you may please suggest me for a PR, if not then well I'll try on different issues !

**Comment by riya-rjha** [comment]
![Screenshot (112)](https://github.com/Innovateninjas/Paws-frontend/assets/112813772/07ab50ef-d6b2-4bb2-be07-867dda7aba3d)

How about this? The above para is with the styles & the one below it is without the styles. 
So can this be counted as an issue? And should I continue with the PR?

**Comment by codewarnab** [comment]
yes looks good but do one this keep the learn more link as before in the mobile and show the button only in bigger screens

**Comment by riya-rjha** [comment]
Okay sure! I'll start working on this then please assign me this issue under the GSSoc label.
